ckan/Dockerfile

```diff
@@ -18,6 +18,12 @@ RUN  pip3 install -e git+https://github.com/GenomicDataInfrastructure/gdi-userpo
 RUN  pip3 install -e git+https://github.com/DataShades/ckanext-oidc-pkce.git@v0.3.1#egg=ckanext-oidc-pkce && \
      pip3 install -r ${APP_DIR}/src/ckanext-oidc-pkce/requirements.txt
 
+RUN  pip3 install -e git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-fairdatapoint.git@main#egg=ckanext-fairdatapoint && \
+     pip3 install -r ${APP_DIR}/src/ckanext-fairdatapoint/requirements.txt
+
+RUN  pip3 install -e git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-civity.git@main#egg=ckanext-civity
```

Comment:
If only `fairdatapointharvester` is needed, can you please remove ckanext-civity from the dependencies?

Reply:
Yes done. see comments slack. Refactor the code for now to make fairdatapoint not depended on it